dist/hypermedia.js

```diff
@@ -150,8 +152,10 @@ angular.module('hypermedia')
     });
 
     ResourceContext.prototype = Object.create(Object.prototype, {
-      constructor: {value: ResourceContext},
+      constructor: {value: ResourceContext}
+    });
 
+    ResourceContext.prototype = HypermediaUtil.defineProperties(ResourceContext.prototype, {
```

Comment:
Do we need to reassign `ResourceContext.prototype here`? I believe `HypermediaUtil.defineProperties` modifies it in-place. Also the other uses of `defineProperties` do not seem to use the return value.